.changeset/honest-jobs-grow.md

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
```suggestion
---
"create-fuels": patch
---
```

Just to confirm I believe we patch `create-fuels`  for changes to templates, right? cc @petertonysmith94